GameData/RSSDateTime/localization.cfg

```diff
@@ -0,0 +1,56 @@
+Localization {
```

Comment:
KSP cfgs usually follow the convention of putting opening braces on a separate line.